src/pages/passengerType.tsx

```diff
@@ -19,9 +26,47 @@ const insetText = 'More passenger types will become available soon';
 interface PassengerTypeProps {
     errors?: ErrorInfo[];
     csrfToken: string;
+    savedGroups: GroupPassengerType[];
 }
 
-const PassengerType = ({ errors = [], csrfToken }: PassengerTypeProps): ReactElement => (
+export const getFieldsets = (
+    errors: ErrorInfo[],
+    savedGroups: GroupPassengerType[],
+): RadioConditionalInputFieldset => ({
+    radioError: errors,
+    heading: {
+        id: 'passenger-types-fieldset',
+        content: '',
+        hidden: true,
+    },
+    radios: PASSENGER_TYPES_WITH_GROUP.map(({ passengerTypeValue: value, passengerTypeDisplay: display }) => ({
+        id: `passenger-type-${value}`,
+        key: value,
+        name: 'passengerType',
+        value,
+        dataAriaControls: 'passenger-type-required-conditional',
+        label: display,
+        type: 'radio',
+        inputType: 'dropdown',
+        ...(value !== GROUP_REUSE_PASSENGER_TYPE
+            ? {}
+            : {
+                  inputHint: {
+                      id: 'choose-time-restriction-hint',
```

Comment:
should be something along the lines of 'choose-saved-group-hint' I think

Reply:
👍